openfecli/commands/quickrun.py

```diff
@@ -28,14 +27,14 @@ def _format_exception(exception) -> str:
     type=click.Path(dir_okay=True, file_okay=False, writable=True,
                     path_type=pathlib.Path),
     help=(
-        "directory to store files in (defaults to current directory)"
+        "Directory in which to store files in (defaults to current directory).\
+        If directory does not exist, it will be created at runtime."
     ),
 )
 @click.option(
     'output', '-o', default=None,
-    type=click.Path(dir_okay=False, file_okay=True, path_type=pathlib.Path),
+    type=click.Path(dir_okay=False, file_okay=False, path_type=pathlib.Path),
```

Comment:
[the `click.File` option](https://click.palletsprojects.com/en/stable/api/#click.File) exists and is neat, but not applicable here since the default behavior relies on the transformation name